themes/openy_themes/openy_carnation/src/scss/layouts/_mob.scss

```diff
@@ -0,0 +1,121 @@
+@media screen and (min-width: 0) and (max-width: 767px) {
+  .search-form-wrapper.col-xs-12 {
+    position: relative;
+
+    .search-input {
+      width: 100%;
+      border: 1px solid white;
+      border-radius: 4px;
+      padding: 5px;
+    }
+  }
+
+  .path-frontpage .paragraph--type--header-banner {
+    margin-bottom: 0 !important;
+  }
+
+  .paragraph--column-in-a-grid .inner {
+    h2 {
+      margin-top: 0;
```

Comment:
can we use spacing https://getbootstrap.com/docs/4.1/utilities/spacing/ instead of stating margins/padding explicitly?

Reply:
Yup, I have used it where I can, I'm going to go ahead and try to remove the _mob.scss and _tab.scss files altogether.